VideoLocker/src/test/java/org/edx/mobile/view/CourseUnitNavigationActivityTest.java

```diff
@@ -397,6 +395,7 @@ public void testUnitFragmentCreation() {
         CourseUnitFragment.HasComponent hasComponent = Mockito.mock(CourseUnitFragment.HasComponent.class);
 
         List<CourseComponent> unitList = new ArrayList<>();
+        List<Class> classesList = new ArrayList<>();
```

Comment:
The `List<Pair<CourseComponent, Class<? extends CourseUnitFragment>>>` doesn't seem like a good idea cuz we can't provide that list to the adapter as it expects a `List<CourseComponent>`.